core/lib/spree/core/engine.rb

```diff
@@ -45,8 +45,16 @@ class Engine < ::Rails::Engine
         Migrations.new(config, engine_name).check
       end
 
-      initializer 'spree.core.subscribe_event_mailer_processor' do
-        Spree::Event::Processors::MailerProcessor.subscribe!
+      # Setup Event Subscribers
+      config.paths.add "app/subscribers", eager_load: true, glob: "**/*_subscriber.rb"
```

Comment:
@elia does this apply only to `app/subscribers` of Solidus core or it will add to the path also subscribers defined into extensions and host applications?

Reply:
@kennyadsl good call, after reading a bunch of rails code and some trial and error I verified I misinterpreted how paths worked. I'll push an update that fixes the behavior for both engine and app code.